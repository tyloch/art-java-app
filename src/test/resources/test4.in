{"type": "Limit", "order": {"direction": "Sell", "id":1, "price": 14, "quantity":20}}
{"type": "Iceberg", "order": {"direction": "Buy", "id":2, "price": 14, "quantity":10, "peak": 5}}


{"type": "Iceberg", "order": {"direction": "Sell", "id":3, "price": 20, "quantity":20, "peak": 5}}
{"type": "Iceberg", "order": {"direction": "Buy", "id":4, "price": 14, "quantity":10, "peak": 5}}

{"type": "Limit", "order": {"direction": "Sell", "id":5, "price": 10, "quantity":200}}
{"type": "Iceberg", "order": {"direction": "Buy", "id":4, "price": 14, "quantity":100, "peak": 5}}