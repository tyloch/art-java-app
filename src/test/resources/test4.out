{"sellOrders":[{"id":1,"price":14,"quantity":20}],"buyOrders":[]}
{"sellOrders":[{"id":1,"price":14,"quantity":10}],"buyOrders":[]}
{"buyOrderId":2,"sellOrderId":1,"price":14,"quantity":5}
{"buyOrderId":2,"sellOrderId":1,"price":14,"quantity":5}